Libraries/TestReport.psm1

```diff
@@ -321,12 +321,7 @@ Class TestSummary
 		if ($this.TestPriority) {
 			$str += "`r`nTest Priority         : $($this.TestPriority)"
 		}
-		if ($global:InitialKernelVersion) {
```

Comment:
Why these logs were moved to summary section ? Any particular reason ?

Reply:
it may testing with various distro, test location,  as I explained to you earlier, more test cases, can be run in the same LISAv2 execution.  never stick to one ARMImageName, so the Kernel version may changes per test case. That's the reason.